pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -65,6 +65,9 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 			queueMap[queue.UID] = queue
 			queues.Push(queue)
 		}
+		if job.PodGroup.Status.Phase == "" {
```

Comment:
Suggest to add some comments here in order not to make confusion for developers later.

Reply:
it  is  modified to `if job.IsPending()` with the if below